<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="2" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="nopCommerceSuite" duration-ms="629" started-at="2019-04-15T17:52:49Z" finished-at="2019-04-15T17:52:50Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="629" started-at="2019-04-15T17:52:49Z" finished-at="2019-04-15T17:52:50Z">
      <class name="tests.RegisterationTestCase">
        <test-method status="PASS" signature="startDriver()[pri:0, instance:tests.RegisterationTestCase@7c457099]" name="startDriver" is-config="true" duration-ms="46031" started-at="2019-04-15T17:52:03Z" finished-at="2019-04-15T17:52:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startDriver -->
        <test-method status="PASS" signature="add_data_to_excel()[pri:1, instance:tests.RegisterationTestCase@7c457099]" name="add_data_to_excel" duration-ms="586" started-at="2019-04-15T17:52:49Z" finished-at="2019-04-15T17:52:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_data_to_excel -->
        <test-method status="SKIP" signature="RegisterNewUserWithCorrectCredentialWithExcel(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:tests.RegisterationTestCase@7c457099]" name="RegisterNewUserWithCorrectCredentialWithExcel" duration-ms="0" started-at="2019-04-15T17:52:50Z" data-provider="excelData" depends-on-methods="tests.RegisterationTestCase.add_data_to_excel" finished-at="2019-04-15T17:52:50Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at data.HandlingExcel.getExcelData(HandlingExcel.java:50)
at tests.RegisterationTestCase.userRegisterData(RegisterationTestCase.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisterNewUserWithCorrectCredentialWithExcel -->
        <test-method status="PASS" signature="killDriver()[pri:0, instance:tests.RegisterationTestCase@7c457099]" name="killDriver" is-config="true" duration-ms="609" started-at="2019-04-15T17:52:50Z" finished-at="2019-04-15T17:52:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killDriver -->
      </class> <!-- tests.RegisterationTestCase -->
    </test> <!-- Test -->
  </suite> <!-- nopCommerceSuite -->
</testng-results>
